go.mod

```diff
@@ -3,32 +3,30 @@ module github.com/bufbuild/knit-go
 go 1.19
 
 require (
-	buf.build/gen/go/bufbuild/knit/bufbuild/connect-go v1.7.0-20230504140941-3dc602456973.1
-	buf.build/gen/go/bufbuild/knit/protocolbuffers/go v1.30.0-20230504140941-3dc602456973.1
-	buf.build/gen/go/bufbuild/reflect/bufbuild/connect-go v1.7.0-20230420145030-e29739ab7137.1
-	github.com/bradfitz/gomemcache v0.0.0-20230124162541-5f7a7d875746
-	github.com/bufbuild/connect-go v1.7.0
-	github.com/bufbuild/connect-grpcreflect-go v1.0.1-0.20230317120624-8e24e9604364
-	github.com/bufbuild/prototransform v0.1.0
+	buf.build/gen/go/bufbuild/knit/connectrpc/go v1.11.0-20230504140941-3dc602456973.1
+	buf.build/gen/go/bufbuild/knit/protocolbuffers/go v1.31.0-20230504140941-3dc602456973.1
+	buf.build/gen/go/bufbuild/reflect/connectrpc/go v1.11.0-20230627113514-3943db27b83c.1
+	connectrpc.com/connect v1.11.0
+	connectrpc.com/grpcreflect v1.2.0
+	github.com/bradfitz/gomemcache v0.0.0-20230611145640-acc696258285
+	github.com/bufbuild/prototransform v0.3.1-0.20230817135718-c86c55ce89fd
```

Comment:
There was no tag yet, so I bumped this manually to pull in https://github.com/bufbuild/prototransform/pull/62